sklearn_xarray/preprocessing.py

```diff
@@ -1298,8 +1305,10 @@ def _transform_var(self, X):
         if self.order is not None:
             stack_dims = self.order
         else:
-            stack_dims = tuple(set(X.dims) - {self.sample_dim})
-
+            if isinstance(self.sample_dim, str):
+                self.sample_dim = [self.sample_dim]
+            stack_dims = tuple(set(X.dims) - set(self.sample_dim))
+            
```

Comment:
These changes should probably be reverted now that there's a dedicated transformer for this

Reply:
Ok not sure how to handle this. Can i leave it to you?

Reply:
Don't worry, if you feel that this is too much at once I can take it from here.

Reply:
Sorry yeah not quite sure what you're looking for.